lib/arena.rb

```diff
@@ -1,2 +1,30 @@
+# require "pry"
+
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize (name)
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator (gladiator)
+    if @gladiators.length < 2
+       @gladiators << gladiator
+    end
+  end
+
+  def fight
+    if @gladiators.length < 2
+        return nil
+    end
+
+    if @gladiators[0].name && @gladiators[1].name
```

Comment:
It looks like this is comparing the names of the gladiator, not the weapons, so this would be why your tests are failing